moveit_demo_nodes/run_move_group/CMakeLists.txt

```diff
@@ -29,7 +29,7 @@ install(TARGETS run_move_group
   EXPORT export_${PROJECT_NAME}
```

Comment:
You should remove this line.  `run_move_group` is not a library and doesn't need to have an export.

Reply:
remove this line: `EXPORT export_${PROJECT_NAME}`

Reply:
```suggestion
```